docs/jwst/background_subtraction/main.rst

```diff
@@ -0,0 +1,183 @@
+Introduction
+------------
+Subtraction of background signal can take several different forms depending on the
+observing mode and the available data. Here we give an overview of the different
+methods that are available, when they can be used, and where they occur in the
+processing flow. Imaging and spectroscopic observations share one method for background
+subtraction, while others are unique to spectroscopic data only. See the documentation
+for the individual steps mentioned here for complete details on how each of them
+function.
+
+Imaging Mode
+------------
+Background subtraction for imaging data is currently only available in two places
+within the calibration pipeline:
+
+1. Image-from-image subtraction during :ref:`calwebb_image2 <calwebb_image2>`
+   processing in the :ref:`background <background_step>` step. The background images
+   come from observations of a dedicated background target.
+2. Local background subtraction for individual sources in the
+   :ref:`calwebb_image3 <calwebb_image3>` pipeline
+   :ref:`source_catalog <source_catalog_step>` step
```

Comment:
Currently in resampling, there's implicit background subtraction happening when we produce the `i2d` files.  I.e. the values computed in `skymatch` get subtracted before drizzling happens, which applies the `skymatch` results, but don't then go back and add back in the initial pedestal.  Is that a bug or feature?

I'm of the opinion that we should probably be adding back in the pedestal value so that the resampled images have the same mean background level as the reference input image.

Reply:
Hmmm, forgot about the background subtraction that can be done in `skymatch` and/or `resampling`. I think it's probably *not* desirable to have `resample` just go ahead and subtract background without it being a controllable option. So I guess I should mention these steps too when I list the possible ways of doing imaging background subtraction.

Reply:
I guess we should perhaps first consult the CalWG to make sure they agree that the `resample` step should not be doing it blindly and automatically. At the very least it should be a user-settable param, just like it is in `skymatch`.

Reply:
Precisely why I'm thinking about it right now.

Of course we don't propagate errors for any background subtraction right now, so it's all "unaccounted" at the moment.

Reply:
OK, could you file a JP ticket asking whether we should turn off background subtraction and assign it to Anton?